Tests/Client/Unit Tests/QueryEncodingTests.swift

```diff
@@ -10,7 +10,7 @@ import XCTest
 @testable import StreamChatClient
 
 final class QueryEncodingTests: XCTestCase {
-    private let channel = Client(apiKey: "test").channel(type: .messaging, id: "general")
+  private let channel = Client(configuration: .init(apiKey: "test")).channel(type: .messaging, id: "general")
```

Comment:
Strange indentation :D

Reply:
Sorry, I always used 2 spaces (like in the Swift standard library). Sometimes my Xcode switches back.